lib/action_cable/channel/broadcasting.rb

```diff
@@ -0,0 +1,21 @@
+module ActionCable
+  module Channel
+    module Broadcasting
+      # Broadcast a hash a unique broadcasting for this <tt>object</tt> in this channel.
```

Comment:
`a hash a unique broadcasting` - what does this mean?

Speaking more generally, I find the above documentation tough to follow.